### PR TITLE
Added bootstrap styles, some custom fonts, half-decent padding

### PR DESCRIPTION
**Pull request by asselstine**
You can see the new site on Netlify:

https://zen-shirley-5f0272.netlify.com/

### CODE REVIEW

**Comment by pet3r-pan** [comment]
Love it!

**Comment by bmann** [comment]
@jpitts @ligi can I get a thumbs up on this? These are theming changes that look good to me.

File: src/index.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-body {
+/* body {
```
Comment (ligi) [inline src/index.css]: wonder why it is commented out and not completely removed

**Review by ligi — APPROVED** [review]
:+1: 

thanks for the change. Would love to see the "commenting out" removed before the merge though.

**Comment by asselstine** [comment]
@ligi This PR was done very quickly :) 

Tonight I'll remove the comments and pull out the off-chain thanks, as @bmann requested.

**Comment by bmann** [comment]
woot woot. Open source PR reviews and issues in action.

Thanks @ligi for taking a look, those are good enhancements. And should go upstream to Giveth at some point, too!

**Comment by asselstine** [comment]
Updated.

**Comment by jpitts** [comment]
Thanks, @asselstine!

**Pull request merged**